packages/expo-video-thumbnails/ios/VideoThumbnailsModule.swift

```diff
@@ -44,20 +40,20 @@ public class VideoThumbnailsModule: Module {
   Saves the image as a file.
   */
   internal func saveImage(image: UIImage, quality: Double) throws -> URL {
-    guard let fileSystem = self.appContext?.fileSystem else {
-      throw Exceptions.FileSystemModuleNotFound()
-    }
-
-    let directory = URL(fileURLWithPath: fileSystem.cachesDirectory).appendingPathComponent("VideoThumbnails")
+    let directory = appContext?.config.cacheDirectory?.appendingPathComponent("VideoThumbnails")
     let fileName = UUID().uuidString.appending(".jpg")
-    let fileUrl = directory.appendingPathComponent(fileName)
+    let fileUrl = directory?.appendingPathComponent(fileName)
 
-    fileSystem.ensureDirExists(withPath: directory.path)
+    FileSystemUtilities.ensureDirExists(at: directory)
 
     guard let data = image.jpegData(compressionQuality: CGFloat(quality)) else {
       throw CorruptedImageDataException()
     }
 
+    guard let fileUrl else {
+      throw ImageWriteFailedException("Unrecognised url \(fileUrl?.path)")
```

Comment:
American English 🤷‍♂️ 
```suggestion
      throw ImageWriteFailedException("Unrecognized url \(fileUrl?.path)")
```